python/feathub/feature_tables/sources/file_system_source.py

```diff
@@ -54,16 +55,20 @@ def __init__(
             properties={
                 "path": path,
             },
+            is_bounded=True,
             data_format=data_format,
             keys=keys,
-            schema=schema,
             timestamp_field=timestamp_field,
             timestamp_format=timestamp_format,
+            schema=schema,
         )
         self.path = path
         self.schema = schema
         self.max_out_of_orderness = max_out_of_orderness
 
+    def get_bounded_view(self) -> TableDescriptor:
```

Comment:
Instead of having every bounded source duplicate this method, would it be simpler to just add a default implementation in the parent class with the following logic:

Return self if is_bounded() == True. Otherwise raise exception.